openquake/commonlib/oqvalidation.py

```diff
@@ -2282,8 +2282,8 @@ def to_ini(key, val):
         return f'{key} = {", ".join(c for c in coords[:-1])}'
     elif key == 'hazard_imtls':
         return f"intensity_measure_types_and_levels = {val}"
-    elif key in ('reqv_ignore_sources', 'poes', 'quantiles',
-                 'source_id', 'source_nodes', 'soil_intensities'):
+    elif key in ('reqv_ignore_sources', 'poes', 'poes_disagg', 'disagg_outputs',
+                 'quantiles', 'source_id', 'source_nodes', 'soil_intensities'):
```

Comment:
Since years, `poes_disagg` is just an alias for `poes`. So, you do not need to add `poes_disagg` here, rather in all of your .ini files change `poes_disagg` with `poes`.

Reply:
Thanks I've just made this change now and repushed